internal/core/metadata/init.go

```diff
@@ -18,8 +18,9 @@ import (
 	"context"
 	"sync"
 
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/container"
 	metadataContainer "github.com/edgexfoundry/edgex-go/internal/core/metadata/container"
```

Comment:
You have two different references to the same package; one aliased, one not.  Recommend removing the aliased one (`metadataContainer`) as doing so would be consistent with similar refactorings in other core services.